app/services/live_config/store.py

```
@@ -15,262 +10,309 @@
-        redis_service = await get_redis_service()
-        if not redis_service:
-            return False
-        return await redis_service.set(
-            f"{FEATURE_FLAGS_PREFIX}{key}", json.dumps(value)
-        )
+        redis = await get_redis_service()
+        client = await redis.get_client()
+
+        raw = await client.get(f"{FEATURE_FLAGS_PREFIX}{key}")
+        return json.loads(raw) if raw else None
     except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)